src/app/tests/suites/certification/Test_TC_SMOKECO_2_2.yaml

```diff
@@ -123,7 +123,7 @@ tests:
       PICS: SMOKECO.S.E00
       command: "readEvent"
       event: "SmokeAlarm"
-      eventNumber: "newEventsOnly"
+      eventNumber: "LastReceivedEventNumber + 1"
```

Comment:
This is changing the test in a way that can be failed by a perfectly spec-compliant device.  Why?

Reply:
This makes the case behavior IDENTICAL with what the code was doing before, but I guess this makes it explicit that maybe we were not doing the right thing somehow. I guess instead of "value" we should have a "lastValue" check or something, either way this does not change the test.

Reply:
OK.